adafruit_logging.py

```diff
@@ -92,6 +92,7 @@ def write(self, buf: str) -> int:
     "StreamHandler",
     "logger_cache",
     "getLogger",
+    "useTimestamp",
```

Comment:
This doesn't look a like a standard CPython API because it isn't here: https://docs.python.org/3/library/logging.html#module-level-functions

How would you do it in CPython? Please match that.